docs/csharp/language-reference/keywords/ref.md

```diff
@@ -89,7 +89,7 @@ For an example, see [A ref returns and ref locals example](#a-ref-returns-and-re
 
 ## Ref locals
 
-A ref local variable is used to refer to values returned using `return ref`.  A ref local variable must be initialized and assigned to a ref return value. Any modifications to the value of the ref local are reflected in the state of the object whose method returned the value by reference.
+A ref local variable is used to refer to values returned using `return ref`.  A ref local variable must be initialized and assigned by a ref return value. Any modifications to the value of the ref local are reflected in the state of the object whose method returned the value by reference.
```

Comment:
The original is actually correct, but could be phrased better.  A better channel would be "A ref local variable cannot be initialized to a non-ref return value.  In other words, the right hand side of the initialization must be a reference."